core/pom.xml

```diff
@@ -31,15 +31,36 @@
   <artifactId>feast-core</artifactId>
 
   <build>
-    <plugins>
-      <plugin>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-maven-plugin</artifactId>
-        <configuration>
-          <skip>false</skip>
-        </configuration>
-      </plugin>
-    </plugins>
+      <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <configuration>
+                  <release>11</release>
+              </configuration>
+          </plugin>
+
+          <plugin>
+              <groupId>org.jacoco</groupId>
+              <artifactId>jacoco-maven-plugin</artifactId>
+          </plugin>
+
+          <plugin>
+              <groupId>org.springframework.boot</groupId>
+              <artifactId>spring-boot-maven-plugin</artifactId>
+              <configuration>
+                  <skip>false</skip>
+              </configuration>
+              <executions>
+                  <execution>
+                      <id>build-info</id>
+                      <goals>
+                          <goal>build-info</goal>
```

Comment:
What is the reason for adding this exclusion?

Reply:
execution not exclusion.  I think it came from master.  I'm not really sure about much in the pom.  Feel free to push changes if you know better.

Reply:
> execution not exclusion. I think it came from master. I'm not really sure about much in the pom. Feel free to push changes if you know better.

Oops, misread.